src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -5,14 +5,14 @@
     <AssemblyName>System.Net.Sockets</AssemblyName>
     <ProjectGuid>{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <EnableWinRT Condition="'$(TargetGroup)' == 'netcore50'">true</EnableWinRT>
+    <EnableWinRT Condition="'$(TargetGroup)' == 'uap'">true</EnableWinRT>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>
-    <AssemblyVersion Condition="'$(TargetGroup)' == 'netcore50'">4.1.1.0</AssemblyVersion>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50'">$(DefineConstants);netcore50</DefineConstants>
+    <AssemblyVersion Condition="'$(TargetGroup)' == 'uap'">4.1.1.0</AssemblyVersion>
```

Comment:
Yeah, I deleted where I noticed it but probably missed a few.  @joperezr feel free to clean up in your work.